manifest.json

```diff
@@ -12,19 +12,19 @@
     },
```

Comment:
Signature for `14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq`:

`WE1pLU0NrmB9UopEvWg3ZELgNgxD9qeR6FnZk6naObLk5izscIX5Nm57+v1I/DVIup3UWdRAvBbh1Y5n/ec7Aw==`